api/src/pbtar_api/app.py

```diff
@@ -27,6 +27,9 @@ def create_app(title, description, version):
     # Configure CORS
     origins = [
         "http://localhost",
+        "http://localhost:3000",
+        "http://0.0.0.0",
+        "http://0.0.0.0:3000",
```

Comment:
Had to make a small change to the API service to configure CORS to accept requests from the frontend.